src/hooks/useAccountBalance.ts

```diff
@@ -88,25 +94,67 @@ export const useAccountBalance = ({
     },
   });
 
+  const { data: accounts } = useAccountGraz({
+    chainId: SUPPORTED_COSMOS_CHAINS,
+    multiChain: true,
+  });
+
```

Comment:
I think this should also be moved to the useAccounts hook with `cosmosAddress` below